src/client/app/services/github.service.ts

```diff
@@ -119,10 +119,21 @@ export class GitHubService {
         let url = `${this._baseUrl}/gists`;
         if (!(id == null)) {
             url += `/${id}`;
-            return this._request.patch<IGist>(url, body, ResponseTypes.JSON, this._headers);
+            return this.gist(id)
+                .flatMap(gist => {
```

Comment:
flatMap is an alias for mergeMap, but we use mergeMap everywhere else. Probably best to keep the name consistent.